<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: center;">
      <th>id</th>
      <th>titre</th>
      <th>pmid</th>
      <th>pmc</th>
      <th>doi</th>
      <th>XMLaccess</th>
      <th>git</th>
      <th>language</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>Tau-typing: a Nextflow pipeline for finding the best phylogenetic markers in the genome for molecular typing of microbial species</td>
      <td>37417952</td>
      <td>10336021</td>
      <td>10.1093/bioinformatics/btad425</td>
      <td>Free</td>
      <td>github.com/hseabolt/tautyping , github.com/tseemann/mlst</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>2</td>
      <td>pgMAP: a pipeline to enable guide RNA read mapping from dual-targeting\nCRISPR screens</td>
      <td>37396617</td>
      <td>10312807</td>
      <td></td>
      <td>No</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>3</td>
      <td>snHiC: a complete and simplified snakemake pipeline for grouped Hi-C data analysis</td>
      <td>37397353</td>
      <td>10307938</td>
      <td>10.1093/bioadv/vbad080</td>
      <td>Free</td>
      <td>github.com/sebastian-gregoricchio/snHiC</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>4</td>
      <td>CRISPR-Analytics (CRISPR-A): A platform for precise analytics and simulations for gene editing</td>
      <td>37253059</td>
      <td>10256225</td>
      <td>10.1371/journal.pcbi.1011137</td>
      <td>Free</td>
      <td>github.com/nanoporetech/pipeline-umi-amplicon , github.com/ocxtal/minialign , github.com/nanoporetech/medaka</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>5</td>
      <td>LENS: Landscape of Effective Neoantigens Software</td>
      <td>37184881</td>
      <td>10246587</td>
      <td>10.1093/bioinformatics/btad322</td>
      <td>Free</td>
      <td></td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>6</td>
      <td>gExcite: a start-to-end framework for single-cell gene expression, hashing, and antibody analysis</td>
      <td>37220897</td>
      <td>10229235</td>
      <td>10.1093/bioinformatics/btad329</td>
      <td>Free</td>
      <td>github.com/ETH-NEXUS/gExcite_pipeline</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>7</td>
      <td>Genofunc: genome annotation and identification of genome features for automated pipelining analysis of virus whole genome sequences</td>
      <td>37254048</td>
      <td>10227794</td>
      <td>10.1186/s12859-023-05356-3</td>
      <td>Free</td>
      <td>github.com/xiaoyu518/genofunc</td>
      <td>Both</td>
    </tr>
    <tr>
      <td>8</td>
      <td>DNA 5-methylcytosine detection and methylation phasing using PacBio circular consensus sequencing</td>
      <td>37422489</td>
      <td>10329642</td>
      <td>10.1038/s41467-023-39784-9</td>
      <td>Free</td>
      <td>github.com/PacificBiosciences/pb-CpG-tools , github.com/PacificBiosciences/ccs , github.com/PengNi/deepsignal2 , github.com/PacificBiosciences/pbmm2 , github.com/human-pangenomics/HG002_Data_Freeze_v1 , github.com/genome-in-a-bottle/giab_data_indexes , github.com/marbl/CHM13 , github.com/PengNi/ccsmeth , github.com/PengNi/ccsmethphase</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>9</td>
      <td>PepGM: a probabilistic graphical model for taxonomic inference of viral proteome samples with associated confidence scores</td>
      <td>37129543</td>
      <td>10182852</td>
      <td>10.1093/bioinformatics/btad289</td>
      <td>Free</td>
      <td>github.com/BAMeScience/PepGM</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>10</td>
      <td>UmetaFlow: an untargeted metabolomics workflow for high-throughput data processing and analysis</td>
      <td>37173725</td>
      <td>10176759</td>
      <td>10.1186/s13321-023-00724-w</td>
      <td>Free</td>
      <td>github.com/biosustain/snakemake_UmetaFlow , github.com/biosustain/pyOpenMS_UmetaFlow , github.com/axelwalter/umetaflow-gui</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>11</td>
      <td>DNAscan2: a versatile, scalable, and user-friendly analysis pipeline for human next-generation sequencing data</td>
      <td>37010501</td>
      <td>10112953</td>
      <td>10.1093/bioinformatics/btad152</td>
      <td>Free</td>
      <td>github.com/KHP-Informatics/DNAscanv2 , github.com/francesca-lucas/ehdn-to-eh , github.com/KHP-Informatics/DNAscanv2_snakemake</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>12</td>
      <td>A modular metagenomics analysis system for integrated multi-step data exploration</td>
      <td>37066359</td>
      <td>10104186</td>
      <td>10.1101/2023.04.09.536171</td>
      <td>Free</td>
      <td>github.com/MetaSUB-CAMP</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>13</td>
      <td>EGAsubmitter: A software to automate submission of nucleic acid sequencing data to the European Genome-phenome Archive</td>
      <td>37063647</td>
      <td>10098081</td>
      <td>10.3389/fbinf.2023.1143014</td>
      <td>Free</td>
      <td>github.com/bioinformaticspolito/EGAsubmitter</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>14</td>
      <td>transXpress: a Snakemake pipeline for streamlined de novo transcriptome assembly and annotation</td>
      <td>37016291</td>
      <td>10074830</td>
      <td>10.1186/s12859-023-05254-8</td>
      <td>Free</td>
      <td>github.com/transXpress/transXpress</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>15</td>
      <td>TIMSCONVERT: a workflow to convert trapped ion mobility data to open data formats</td>
      <td>35758608</td>
      <td>9991885</td>
      <td>10.1093/bioinformatics/btac419</td>
      <td>No</td>
      <td></td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>16</td>
      <td>polishCLR: A Nextflow Workflow for Polishing PacBio CLR Genome Assemblies</td>
      <td>36792366</td>
      <td>9985148</td>
      <td>10.1093/gbe/evad020</td>
      <td>Free</td>
      <td>github.com/isugifNF/polishCLR</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>17</td>
      <td>Reproducible evaluation of transposable element detectors with McClintock 2 guides accurate inference of Ty insertion patterns in yeast</td>
      <td>36824955</td>
      <td>9948991</td>
      <td>10.1101/2023.02.13.528343</td>
      <td>Free</td>
      <td>github.com/bergmanlab/mcclintock/ , github.com/conda/conda , github.com/FelixKrueger/TrimGalore , github.com/Snakemake-Profiles/ , github.com/bergmanlab/mcclintock/blob/master/test/sac_cer_TE_seqs , github.com/bergmanlab/mcclintock/blob/master/test/</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>18</td>
      <td>circRNA-sponging: a pipeline for extensive analysis of circRNA expression and their role in miRNA sponging</td>
      <td>36789427</td>
      <td>9928029</td>
      <td>10.1101/2023.01.19.524495</td>
      <td>Free</td>
      <td>github.com/biomedbigdata/circRNA-sponging</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>19</td>
      <td>Snekmer: a scalable pipeline for protein sequence fingerprinting based on amino acid recoding</td>
      <td>36789294</td>
      <td>9913046</td>
      <td>10.1093/bioadv/vbad005</td>
      <td>Free</td>
      <td>github.com/PNNL-CompBio/Snekmer</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>20</td>
      <td>Mapache: a flexible pipeline to map ancient DNA</td>
      <td>36637197</td>
      <td>9901408</td>
      <td>10.1093/bioinformatics/btad028</td>
      <td>Free</td>
      <td>github.com/sneuensc/mapache , github.com/sneuensc/mapache/wiki</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>21</td>
      <td>HISS: Snakemake-based workflows for performing SMRT-RenSeq assembly, AgRenSeq and dRenSeq for the discovery of novel plant disease resistance genes</td>
      <td>37198529</td>
      <td>10193785</td>
      <td>10.1186/s12859-023-05335-8</td>
      <td>Free</td>
      <td>github.com/SwiftSeal/HISS</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>22</td>
      <td>VirPipe: an easy-to-use and customizable pipeline for detecting viral genomes from Nanopore sequencing</td>
      <td>37129547</td>
      <td>10191607</td>
      <td>10.1093/bioinformatics/btad293</td>
      <td>Free</td>
      <td>github.com/KijinKims/VirPipe</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>23</td>
      <td>A2TEA: Identifying trait-specific evolutionary adaptations</td>
      <td>37224329</td>
      <td>10186066</td>
      <td>10.12688/f1000research.126463.2</td>
      <td>Free</td>
      <td>github.com/tgstoecker/A2TEA , github.com/groupschoof/AHRD , github.com/mamba-org/mamba</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>24</td>
      <td>A computationally-enhanced hiCLIP atlas reveals Staufen1-RNA binding features and links 3′ UTR structure to RNA metabolism</td>
      <td>37013995</td>
      <td>10164587</td>
      <td>10.1093/nar/gkad221</td>
      <td>Free</td>
      <td>github.com/ulelab/icount-mini , github.com/qczhang/icSHAPE , github.com/caochch/RIC-seq , github.com/amchakra/tosca , github.com/luslab/comp-hiclip</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>25</td>
      <td>RNAcontacts: A Pipeline for Predicting Contacts from RNA Proximity Ligation Assays</td>
      <td>37153509</td>
      <td>10154773</td>
      <td>10.32607/actanaturae.11893</td>
      <td>Free</td>
      <td>github.com/smargasyuk/</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>26</td>
      <td>The GEN-ERA toolbox: unified and reproducible workflows for research in microbial genomics</td>
      <td>37036103</td>
      <td>10084500</td>
      <td>10.1093/gigascience/giad022</td>
      <td>Free</td>
      <td>github.com/Lcornet/GENERA</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>27</td>
      <td>SnakeMAGs: a simple, efficient, flexible and scalable workflow to reconstruct prokaryotic genomes from metagenomes</td>
      <td>36875992</td>
      <td>9978240</td>
      <td>10.12688/f1000research.128091.2</td>
      <td>Free</td>
      <td>github.com/Nachida08/SnakeMAGs</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>28</td>
      <td>Performance of methods for SARS-CoV-2 variant detection and abundance estimation within mixed population samples</td>
      <td>36721781</td>
      <td>9884472</td>
      <td>10.7717/peerj.14596</td>
      <td>Free</td>
      <td>github.com/cov-lineages/constellations/tree/main/constellations/definitions , github.com/CDCgov/datasets-sars-cov-2/blob/master/datasets/sars-cov-2-voivoc , github.com/CFSAN-Biostatistics/ww_simulations , github.com/CFSAN-Biostatistics/C-WAP</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>29</td>
      <td>nf-core/circrna: a portable workflow for the quantification, miRNA target prediction and differential expression analysis of circular RNAs</td>
      <td>36694127</td>
      <td>9875403</td>
      <td>10.1186/s12859-022-05125-8</td>
      <td>Free</td>
      <td>github.com/nf-core/circrna , github.com/nf-core/configs , github.com/BarryDigby/circRNA_simu</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>30</td>
      <td>What the Phage: a scalable workflow for the identification and analysis of phage sequences</td>
      <td>36399058</td>
      <td>9673492</td>
      <td>10.1093/gigascience/giac110</td>
      <td>Free</td>
      <td>github.com/replikation/What_the_Phage</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>31</td>
      <td>RetroSnake: A modular pipeline to detect human endogenous retroviruses in genome sequencing data</td>
      <td>36339261</td>
      <td>9626663</td>
      <td>10.1016/j.isci.2022.105289</td>
      <td>Free</td>
      <td>github.com/KHP-Informatics/RetroSnake</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>32</td>
      <td>QuasiFlow: a Nextflow pipeline for analysis of NGS-based HIV-1 drug resistance data</td>
      <td>36699347</td>
      <td>9722223</td>
      <td>10.1093/bioadv/vbac089</td>
      <td>Free</td>
      <td>github.com/AlfredUg/QuasiFlow</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>33</td>
      <td>TOSCA: an automated Tumor Only Somatic CAlling workflow for somatic mutation detection without matched normal samples</td>
      <td>36699358</td>
      <td>9710689</td>
      <td>10.1093/bioadv/vbac070</td>
      <td>Free</td>
      <td>github.com/mdelcorvo/TOSCA</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>34</td>
      <td>TransFlow: a Snakemake workflow for transmission analysis of Mycobacterium tuberculosis whole-genome sequencing data</td>
      <td>36469333</td>
      <td>9825751</td>
      <td>10.1093/bioinformatics/btac785</td>
      <td>Free</td>
      <td>github.com/cvn001/transflow</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>35</td>
      <td>Hex Me If You Can</td>
      <td>36636106</td>
      <td>9828569</td>
      <td>10.1111/cgf.14608</td>
      <td>Free</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>36</td>
      <td>Comparison of Metagenomics and Metatranscriptomics Tools: A Guide to Making the Right Choice</td>
      <td>36553546</td>
      <td>9777648</td>
      <td>10.3390/genes13122280</td>
      <td>Free</td>
      <td>github.com/BioinfoIPBLN/16S-Metatranscriptomic-Analysis</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>37</td>
      <td>hgtseq: A Standard Pipeline to Study Horizontal Gene Transfer</td>
      <td>36498841</td>
      <td>9738810</td>
      <td>10.3390/ijms232314512</td>
      <td>Free</td>
      <td>github.com/marcelm/cutadapt/ , github.com/pezmaster31/bamtools , github.com/nf-core/hgtseq</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>38</td>
      <td>annotate_my_genomes: an easy-to-use pipeline to improve genome annotation and uncover neglected genes by hybrid RNA sequencing</td>
      <td>36472574</td>
      <td>9724561</td>
      <td>10.1093/gigascience/giac099</td>
      <td>Free</td>
      <td>github.com/cfarkas/annotate_my_genomes , github.com/TransDecoder/TransDecoder , github.com/ben-lerch/IsoSeq-3 , github.com/yunhaowang/IsoSeqSim , github.com/cfarkas/annotate_my_genomes/wiki , github.com/enormandeau/gawn , github.com/deeptools/pyGenomeTracks , github.com/cfarkas/annotate_my_genomes/wiki/annotate_my_genomes-benchmarking</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>39</td>
      <td>MetaGT: A pipeline for de novo assembly of metatranscriptomes with the aid of metagenomic data</td>
      <td>36386613</td>
      <td>9651917</td>
      <td>10.3389/fmicb.2022.981458</td>
      <td>Free</td>
      <td>github.com/ablab/metaGT , github.com/TransDecoder/TransDecoder</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>40</td>
      <td>Fusarium oxysporum effector clustering version 2: An updated pipeline to infer host range</td>
      <td>36340405</td>
      <td>9627151</td>
      <td>10.3389/fpls.2022.1012688</td>
      <td>Free</td>
      <td>github.com/pvdam3/FoEC2 , github.com/pvdam3/FoEC</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>41</td>
      <td>MetaWorks: A flexible, scalable bioinformatic pipeline for high-throughput multi-marker biodiversity assessments</td>
      <td>36174014</td>
      <td>9521933</td>
      <td>10.1371/journal.pone.0274260</td>
      <td>Free</td>
      <td>github.com/terrimporter/MetaWorks</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>42</td>
      <td>scATACpipe: A nextflow pipeline for comprehensive and reproducible analyses of single cell ATAC-seq data</td>
      <td>36238687</td>
      <td>9551270</td>
      <td>10.3389/fcell.2022.981859</td>
      <td>Free</td>
      <td>github.com/hukai916/scATACpipe , github.com/hukai916/scATACpipe/blob/main/docs/usage , github.com/hukai916/scATACpipe/blob/main/docs/template_of_method , github.com/10Xgenomics/cellranger , github.com/haibol2016/PBMC_scRNAseq_analysis , github.com/hukai916/scATACpipe/tree/main</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>43</td>
      <td>NanoRTax, a real-time pipeline for taxonomic and diversity analysis of nanopore 16S rRNA amplicon sequencing data</td>
      <td>36212537</td>
      <td>9522874</td>
      <td>10.1016/j.csbj.2022.09.024</td>
      <td>Free</td>
      <td>github.com/genomicsITER/NanoRTax</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>44</td>
      <td>MetaPhage: an Automated Pipeline for Analyzing, Annotating, and Classifying Bacteriophages in Metagenomics Sequencing Data</td>
      <td>36069454</td>
      <td>9599279</td>
      <td>10.1128/msystems.00741-22</td>
      <td>Free</td>
      <td>github.com/MattiaPandolfoVR/MetaPhage , github.com/lazzarigioele/graphanalyzer , github.com/s4hts/HTStream</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>45</td>
      <td>TMBur: a distributable tumor mutation burden approach for whole genome sequencing</td>
      <td>36071521</td>
      <td>9450342</td>
      <td>10.1186/s12920-022-01348-z</td>
      <td>Free</td>
      <td>github.com/bcgsc/TMBur</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>46</td>
      <td>A bioinformatic-assisted workflow for genome-wide identification of ncRNAs</td>
      <td>35979446</td>
      <td>9376865</td>
      <td>10.1093/nargab/lqac059</td>
      <td>Free</td>
      <td>github.com/brummetheus/pinc</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>47</td>
      <td>Tourmaline: A containerized workflow for rapid and iterable amplicon sequence analysis using QIIME 2 and Snakemake</td>
      <td>35902092</td>
      <td>9334028</td>
      <td>10.1093/gigascience/giac066</td>
      <td>Free</td>
      <td>github.com/aomlomics/tourmaline</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>48</td>
      <td>iCOMIC: a graphical interface-driven bioinformatics pipeline for analyzing cancer omics data</td>
      <td>35899080</td>
      <td>9310080</td>
      <td>10.1093/nargab/lqac053</td>
      <td>Free</td>
      <td>github.com/broadinstitute/picard , github.com/RamanLab/iCOMIC/blob/main/icomic_env , github.com/RamanLab/iCOMIC</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>49</td>
      <td>Spacemake: processing and analysis of large-scale spatial transcriptomics data</td>
      <td>35852420</td>
      <td>9295369</td>
      <td>10.1093/gigascience/giac064</td>
      <td>Free</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>50</td>
      <td>Snaq: A Dynamic Snakemake Pipeline for Microbiome Data Analysis With QIIME2</td>
      <td>36304319</td>
      <td>9580898</td>
      <td>10.3389/fbinf.2022.893933</td>
      <td>Free</td>
      <td>github.com/qiime2/q2galaxy , github.com/attayeb/snaq</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>51</td>
      <td>Implementing the reuse of public DIA proteomics datasets: from the PRIDE database to Expression Atlas</td>
      <td>35701420</td>
      <td>9197839</td>
      <td>10.1038/s41597-022-01380-9</td>
      <td>Free</td>
      <td>github.com/PRIDE-reanalysis/DIA-reanalysis</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>52</td>
      <td>GenErode: a bioinformatics pipeline to investigate genome erosion in endangered and extinct species</td>
      <td>35698034</td>
      <td>9195343</td>
      <td>10.1186/s12859-022-04757-0</td>
      <td>Free</td>
      <td>github.com/NBISweden/GenErode , github.com/broadinstitute/picard , github.com/Snakemake-Profiles/ , github.com/NBISweden/GenErode/wiki/ , github.com/NBISweden/GenErode/</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>53</td>
      <td>Detection and Validation of Circular DNA Fragments Using Nanopore Sequencing</td>
      <td>35711922</td>
      <td>9195511</td>
      <td>10.3389/fgene.2022.867018</td>
      <td>Free</td>
      <td>github.com/snakemake-workflows/cyrcular-calling</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>54</td>
      <td>Exodus: sequencing-based pipeline for quantification of pooled variants</td>
      <td>35551337</td>
      <td>9191209</td>
      <td>10.1093/bioinformatics/btac319</td>
      <td>Free</td>
      <td>github.com/ilyavs/exodus</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>55</td>
      <td>BioUML—towards a universal research platform</td>
      <td>35536253</td>
      <td>9252820</td>
      <td>10.1093/nar/gkac286</td>
      <td>Free</td>
      <td>github.com/Biosoft-ru/pybiouml , github.com/mozilla/rhino</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>56</td>
      <td>GEMmaker: process massive RNA-seq datasets on heterogeneous computational infrastructure</td>
      <td>35501696</td>
      <td>9063052</td>
      <td>10.1186/s12859-022-04629-7</td>
      <td>Free</td>
      <td>github.com/SystemsGenetics/GEMmaker</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>57</td>
      <td>Findzx: an automated pipeline for detecting and visualising sex chromosomes using whole-genome sequencing data</td>
      <td>35477344</td>
      <td>9044604</td>
      <td>10.1186/s12864-022-08432-9</td>
      <td>Free</td>
      <td>github.com/hsigeman/findZX</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>58</td>
      <td>A bioinformatic pipeline for simulating viral integration data</td>
      <td>35496474</td>
      <td>9046613</td>
      <td>10.1016/j.dib.2022.108161</td>
      <td>Free</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>59</td>
      <td>MEDUSA: A Pipeline for Sensitive Taxonomic Classification and Flexible Functional Annotation of Metagenomic Shotgun Sequences</td>
      <td>35330728</td>
      <td>8940201</td>
      <td>10.3389/fgene.2022.814437</td>
      <td>Free</td>
      <td>github.com/dalmolingroup/MEDUSA_supplements</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>60</td>
      <td>epiGBS2: Improvements and evaluation of highly multiplexed, epiGBS‐based reduced representation bisulfite sequencing</td>
      <td>35178872</td>
      <td>9311447</td>
      <td>10.1111/1755-0998.13597</td>
      <td>Free</td>
      <td>github.com/nioo , github.com/nugentechnologies/NuMetRRBS/blob/master/trimRRBSdiversityAdaptCustomers , github.com/MaartenPostuma/epiGBS</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>61</td>
      <td>Enhanced protein isoform characterization through long-read proteogenomics</td>
      <td>35241129</td>
      <td>8892804</td>
      <td>10.1186/s13059-022-02624-y</td>
      <td>Free</td>
      <td>github.com/smith-chem-wisc/MetaMorpheus/wiki/Protein-Parsimony- , github.com/Magdoll/cDNA_Cupcake , github.com/smith-chem-wisc/MetaMorpheus/tree/LongReadProteogenomics , github.com/sheynkman-lab/Long-Read-Proteogenomics , github.com/sheynkman-lab/Long-Read-Proteogenomics/wiki , github.com/sheynkman-lab/Long-Read-Proteogenomics-Analysis , github.com/sheynkman-lab/Long-Read-Proteogenomics/main , github.com/sheynkman-lab/Long-Read-Proteogenomics/blob/main/README , github.com/sheynkman-lab/Long-Read-Proteogenomics/wiki/Pipeline-Processes , github.com/sheynkman-lab/Long-Read-Proteogenomics/wiki/Pipeline-Vignette</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>62</td>
      <td>CIRCprimerXL: Convenient and High-Throughput PCR Primer Design for Circular RNA Quantification</td>
      <td>36304334</td>
      <td>9580850</td>
      <td>10.3389/fbinf.2022.834655</td>
      <td>Free</td>
      <td>github.com/OncoRNALab/CIRCprimerXL , github.com/ekg/fastahack</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>63</td>
      <td>Topological approximate Bayesian computation for parameter inference of an angiogenesis model</td>
      <td>35191485</td>
      <td>9048691</td>
      <td>10.1093/bioinformatics/btac118</td>
      <td>Free</td>
      <td>github.com/tt104/tabc_angio</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>64</td>
      <td>Phylodynamic Inference of Bacterial Outbreak Parameters Using Nanopore Sequencing</td>
      <td>35171290</td>
      <td>8963328</td>
      <td>10.1093/molbev/msac040</td>
      <td>Free</td>
      <td>github.com/np-core/np-assembly , github.com/tseemann/shovill , github.com/esteinig/sccion , github.com/wwood/CoverM , github.com/nanoporetech/medaka , github.com/tseemann/snippy , github.com/np-core/nanopath</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>65</td>
      <td>riboviz 2: a flexible and robust ribosome profiling data analysis and visualization workflow</td>
      <td>35157051</td>
      <td>9004635</td>
      <td>10.1093/bioinformatics/btac093</td>
      <td>Free</td>
      <td>github.com/riboviz/riboviz , github.com/riboviz/example-datasets</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>66</td>
      <td>SWAAT Bioinformatics Workflow for Protein Structure-Based Annotation of ADME Gene Variants</td>
      <td>35207751</td>
      <td>8875676</td>
      <td>10.3390/jpm12020263</td>
      <td>Free</td>
      <td>github.com/ElofssonLab/PRODRES , github.com/hothman/SWAAT/tree/master/auxiliary_wf , github.com/hothman/SWAAT/tree/master/database , github.com/hothman/SWAAT</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>67</td>
      <td>Side-by-Side Comparison of Post-Entry Quarantine and High Throughput Sequencing Methods for Virus and Viroid Diagnosis</td>
      <td>35205129</td>
      <td>8868628</td>
      <td>10.3390/biology11020263</td>
      <td>Free</td>
      <td>github.com/schmidda/blast-tools , github.com/eresearchqut/virreport</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>68</td>
      <td>nf-core/mag: a best-practice pipeline for metagenome hybrid assembly and binning</td>
      <td>35118380</td>
      <td>8808542</td>
      <td>10.1093/nargab/lqac007</td>
      <td>Free</td>
      <td>github.com/nf-core/mag , github.com/nf-core/modules , github.com/rrwick/Porechop , github.com/rrwick/Filtlong</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>69</td>
      <td>DENTIST—using long reads for closing assembly gaps at high accuracy</td>
      <td>35077539</td>
      <td>8848313</td>
      <td>10.1093/gigascience/giab100</td>
      <td>Free</td>
      <td>github.com/a-ludi/dentist</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>70</td>
      <td>StainedGlass: interactive visualization of massive tandem repeat structures with identity heatmaps</td>
      <td>35020798</td>
      <td>8963321</td>
      <td>10.1093/bioinformatics/btac018</td>
      <td>No</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>71</td>
      <td>nf-rnaSeqCount: A Nextflow pipeline for obtaining raw read counts from RNA-seq data</td>
      <td>35574063</td>
      <td>9097006</td>
      <td>10.18489/sacj.v33i2.830</td>
      <td>Free</td>
      <td>github.com/ , github.com/phelelani/nf-rnaSeqCount</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>72</td>
      <td>Identifying cancer pathway dysregulations using differential causal effects</td>
      <td>34927666</td>
      <td>8896597</td>
      <td>10.1093/bioinformatics/btab847</td>
      <td>Free</td>
      <td>github.com/cbg-ethz/dce , github.com/dirmeier/ggpixel</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>73</td>
      <td>Clones and Clusters of Antimicrobial-Resistant Klebsiella From Southwestern Nigeria</td>
      <td>34850837</td>
      <td>8634535</td>
      <td>10.1093/cid/ciab769</td>
      <td>Free</td>
      <td>github.com/katholt/Kleborate</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>74</td>
      <td>snpQT: flexible, reproducible, and comprehensive quality control and imputation of genomic data</td>
      <td>34900230</td>
      <td>8637247</td>
      <td>10.12688/f1000research.53821.2</td>
      <td>Free</td>
      <td>github.com/nebfield/snpQT , github.com/biocore-ntnu/snpflip , github.com/nebfield/snpQT/releases</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>75</td>
      <td>Comprehensive Analysis of Large-Scale Transcriptomes from Multiple Cancer Types</td>
      <td>34946814</td>
      <td>8701385</td>
      <td>10.3390/genes12121865</td>
      <td>Free</td>
      <td>github.com/nongbaoting/PipeOne , github.com/gpertea/gffcompare , github.com/suhrig/arriba</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>76</td>
      <td>nextNEOpi: a comprehensive pipeline for computational neoantigen prediction</td>
      <td>34788790</td>
      <td>8796378</td>
      <td>10.1093/bioinformatics/btab759</td>
      <td>Free</td>
      <td>github.com/icbi-lab/nextNEOpi</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>77</td>
      <td>Design considerations for workflow management systems use in production genomics research and the clinic</td>
      <td>34737383</td>
      <td>8569008</td>
      <td>10.1038/s41598-021-99288-8</td>
      <td>Free</td>
      <td>github.com/dnanexus/dxWDL , github.com/dnanexus/dx-cwl , github.com/common-workflow-language/cwl-ex , github.com/ohsu-comp-bio/funnel , github.com/ncsa/MayomicsVC/tree/dev-gatk</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>78</td>
      <td>ORPER: A Workflow for Constrained SSU rRNA Phylogenies</td>
      <td>34828348</td>
      <td>8623055</td>
      <td>10.3390/genes12111741</td>
      <td>Free</td>
      <td>github.com/Lcornet/ORPER , github.com/tseemann/barrnap</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>79</td>
      <td>FA-nf: A Functional Annotation Pipeline for Proteins from Non-Model Organisms Implemented in Nextflow</td>
      <td>34681040</td>
      <td>8535801</td>
      <td>10.3390/genes12101645</td>
      <td>Free</td>
      <td>github.com/guigolab/FA-nf , github.com/guigolab/FA-nf/releases , github.com/toniher/biomirror/ , github.com/guigolab/FA-nf/tree/master/dataset</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>80</td>
      <td>Characterisation of tumour microenvironment remodelling following oncogene inhibition in preclinical studies with imaging mass cytometry</td>
      <td>34625563</td>
      <td>8501076</td>
      <td>10.1038/s41467-021-26214-x</td>
      <td>Free</td>
      <td>github.com/BodenmillerGroup/ImcPluginsCP , github.com/BodenmillerGroup/imctools , github.com/nf-core/imcyto , github.com/BodenmillerGroup/neighbouRhood , github.com/FrancisCrickInstitute/vanMaldegem_Valand_2021</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>81</td>
      <td>MuWU: Mutant-seq library analysis and annotation</td>
      <td>34586393</td>
      <td>8756183</td>
      <td>10.1093/bioinformatics/btab679</td>
      <td>Free</td>
      <td>github.com/tgstoecker/MuWU</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>82</td>
      <td>Genetic demultiplexing of pooled single-cell RNA-sequencing samples in cancer facilitates effective experimental design</td>
      <td>34553212</td>
      <td>8458035</td>
      <td>10.1093/gigascience/giab062</td>
      <td>Free</td>
      <td>github.com/lmweber/snp-dmx-cancer</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>83</td>
      <td>VGEA: an RNA viral assembly toolkit</td>
      <td>34567846</td>
      <td>8428259</td>
      <td>10.7717/peerj.12129</td>
      <td>Free</td>
      <td>github.com/pauloluniyi/VGEA , github.com/broadinstitute/viral-ngs , github.com/cbg-ethz/V-pipe/wiki/advanced , github.com/pauloluniyi/VGEA/blob/master/README</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>84</td>
      <td>ExOrthist: a tool to infer exon orthologies at any evolutionary distance</td>
      <td>34416914</td>
      <td>8379844</td>
      <td>10.1186/s13059-021-02441-9</td>
      <td>Free</td>
      <td>github.com/biocorecrg/ExOrthist</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>85</td>
      <td>Large-scale quality assessment of prokaryotic genomes with metashot/prok-quality</td>
      <td>35136576</td>
      <td>8804904</td>
      <td>10.12688/f1000research.54418.1</td>
      <td>Free</td>
      <td>github.com/Ecogenomics/CheckM/wiki/Genome-Quality-Commands , github.com/Ecogenomics/CheckM/issues/118 , github.com/metashot/prok-quality , github.com/metashot/docker</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>86</td>
      <td>nf-LO: A Scalable, Containerized Workflow for Genome-to-Genome Lift Over</td>
      <td>34383887</td>
      <td>8412297</td>
      <td>10.1093/gbe/evab183</td>
      <td>Free</td>
      <td>github.com/evotools/nf-LO</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>87</td>
      <td>ppx: Programmatic access to proteomics data repositories</td>
      <td>34342226</td>
      <td>8457024</td>
      <td>10.1021/acs.jproteome.1c00454</td>
      <td>Free</td>
      <td>github.com/wfondrie/ppx , github.com/Noble-Lab/ppx-workflow</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>88</td>
      <td>poreCov-An Easy to Use, Fast, and Robust Workflow for SARS-CoV-2 Genome Reconstruction via Nanopore Sequencing</td>
      <td>34394197</td>
      <td>8355734</td>
      <td>10.3389/fgene.2021.711437</td>
      <td>Free</td>
      <td>github.com/replikation/poreCov , github.com/nf-core/viralrecon , github.com/connor-lab/ncov2019-artic-nf , github.com/artic-network/fieldbioinformatics , github.com/nanoporetech/medaka , github.com/jts/nanopolish , github.com/cov-lineages/pangolin</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>89</td>
      <td>BIGwas: Single-command quality control and association testing for multi-cohort and biobank-scale GWAS/PheWAS data</td>
      <td>34184051</td>
      <td>8239664</td>
      <td>10.1093/gigascience/giab047</td>
      <td>Free</td>
      <td>github.com/ikmb/gwas-qc , github.com/ikmb/gwas-assoc , github.com/ikmb/chipmatch , github.com/ikmb/genericplotter</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>90</td>
      <td>MicroPIPE: validating an end-to-end workflow for high-quality complete bacterial genome construction</td>
      <td>34172000</td>
      <td>8235852</td>
      <td>10.1186/s12864-021-07767-z</td>
      <td>Free</td>
      <td>github.com/BeatsonLab-MicrobialGenomics/micropipe , github.com/rrwick/Porechop , github.com/rrwick/Filtlong , github.com/mdcao/japsa/ , github.com/lbcb-sci/raven , github.com/chanzuckerberg/shasta/blob/master/conf/Nanopore-Dec2019 , github.com/nanoporetech/pomoxis , github.com/nanoporetech/qcat , github.com/rrwick/Trycycler , github.com/fenderglass/Flye/blob/flye/docs/USAGE</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>91</td>
      <td>Systematic benchmarking of tools for CpG methylation detection from nanopore sequencing</td>
      <td>34103501</td>
      <td>8187371</td>
      <td>10.1038/s41467-021-23778-6</td>
      <td>Free</td>
      <td>github.com/comprna/METEORE , github.com/kpalin/gcf52ref</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>92</td>
      <td>DetectIS: a pipeline to rapidly detect exogenous DNA integration sites using DNA or RNA paired-end sequencing data</td>
      <td>33978747</td>
      <td>9502153</td>
      <td>10.1093/bioinformatics/btab366</td>
      <td>Free</td>
      <td>github.com/AstraZeneca/detectIS</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>93</td>
      <td>The Dockstore: enhancing a community platform for sharing reproducible and accessible computational protocols</td>
      <td>33978761</td>
      <td>8218198</td>
      <td>10.1093/nar/gkab346</td>
      <td>Free</td>
      <td>github.com/dockstore/dockstore</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>94</td>
      <td>Species-Specific Quality Control, Assembly and Contamination Detection in Microbial Isolate Sequences with AQUAMIS</td>
      <td>33926025</td>
      <td>8145556</td>
      <td>10.3390/genes12050644</td>
      <td>Free</td>
      <td>github.com/OLC-Bioinformatics/ConFindr/tree/master/confindr_src , github.com/tseemann/mlst , github.com/tseemann/cgmlst-dists , github.com/apightling/contamination</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>95</td>
      <td>Sustainable data analysis with Snakemake</td>
      <td>34035898</td>
      <td>8114187</td>
      <td>10.12688/f1000research.29032.2</td>
      <td>Free</td>
      <td>github.com/pditommaso/awesome-pipeline , github.com/features/actions , github.com/snakemake/snakefmt , github.com/snakemake/snakemakegithub-action , github.com/snakemake-profiles/doc , github.com/snakemake/snakemake</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>96</td>
      <td>Testcrosses are an efficient strategy for identifying cis-regulatory variation: Bayesian analysis of allele-specific expression (BayesASE)</td>
      <td>33772539</td>
      <td>8104932</td>
      <td>10.1093/g3journal/jkab096</td>
      <td>Free</td>
      <td>github.com/McIntyre-Lab/BayesASE</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>97</td>
      <td>WASP: a versatile, web-accessible single cell RNA-Seq processing platform</td>
      <td>33736596</td>
      <td>7977290</td>
      <td>10.1186/s12864-021-07469-6</td>
      <td>Free</td>
      <td>github.com/andreashoek/wasp</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>98</td>
      <td>Reproducible, portable, and efficient ancient genome reconstruction with nf-core/eager</td>
      <td>33777521</td>
      <td>7977378</td>
      <td>10.7717/peerj.10947</td>
      <td>Free</td>
      <td>github.com/stschiff/sequenceTools , github.com/conda-forge , github.com/nf-core/eager</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>99</td>
      <td>PopInf: An Approach for Reproducibly Visualizing and Assigning Population Affiliation in Genomic Samples of Uncertain Origin</td>
      <td>33074720</td>
      <td>7994427</td>
      <td>10.1089/cmb.2019.0434</td>
      <td>No</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>100</td>
      <td>nf-gwas-pipeline: A Nextflow Genome-Wide Association Study Pipeline</td>
      <td>35647481</td>
      <td>9137404</td>
      <td>10.21105/joss.02957</td>
      <td>Free</td>
      <td>github.com/montilab/nf-gwas-pipeline</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>101</td>
      <td>Using prototyping to choose a bioinformatics workflow management system</td>
      <td>33630841</td>
      <td>7906312</td>
      <td>10.1371/journal.pcbi.1008622</td>
      <td>Free</td>
      <td>github.com/snakemake/snakemake , github.com/common-workflow-language/cwltool , github.com/DataBiosphere/toil , github.com/nextflow-io/nextflow , github.com/riboviz/workflows , github.com/riboviz/riboviz , github.com/spotify/luigi</td>
      <td>Both</td>
    </tr>
    <tr>
      <td>102</td>
      <td>Metagenomics workflow for hybrid assembly, differential coverage binning, metatranscriptomics and pathway analysis (MUFFIN)</td>
      <td>33561126</td>
      <td>7899367</td>
      <td>10.1371/journal.pcbi.1008716</td>
      <td>Free</td>
      <td>github.com/RVanDamme/MUFFIN , github.com/nextflow-io/nextflow/releases</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>103</td>
      <td>lncEvo: automated identification and conservation study of long noncoding RNAs</td>
      <td>33563213</td>
      <td>7871587</td>
      <td>10.1186/s12859-021-03991-2</td>
      <td>Free</td>
      <td>github.com/alexdobin/STAR/blob/master/doc/STARmanual , github.com/ENCODE-DCC/kentUtils , github.com/fbdesignpro/sweetviz , github.com/wwood/ena-fast-download , github.com/mcfrith/last-genome-alignments</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>104</td>
      <td>RNAflow: An Effective and Simple RNA-Seq Differential Gene Expression Pipeline Using Nextflow</td>
      <td>33322033</td>
      <td>7763471</td>
      <td>10.3390/genes11121487</td>
      <td>Free</td>
      <td>github.com/kevinblighe/EnhancedVolcano , github.com/kassambara/ggpubr , github.com/hoelzer-lab/rnaflow</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>105</td>
      <td>Dadasnake, a Snakemake implementation of DADA2 to process amplicon sequencing data for microbial ecology</td>
      <td>33252655</td>
      <td>7702218</td>
      <td>10.1093/gigascience/giaa135</td>
      <td>Free</td>
      <td>github.com/a-h-b/dadasnake</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>106</td>
      <td>Accurate and Efficient KIR Gene and Haplotype Inference From Genome Sequencing Reads With Novel K-mer Signatures</td>
      <td>33324401</td>
      <td>7727328</td>
      <td>10.3389/fimmu.2020.583013</td>
      <td>Free</td>
      <td>github.com/droeatumn/kpi , github.com/droeatumn/kpi/tree/master/input</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>107</td>
      <td>Cascabel: A Scalable and Versatile Amplicon Sequence Data Analysis Pipeline Delivering Reproducible and Documented Results</td>
      <td>33329686</td>
      <td>7718033</td>
      <td>10.3389/fgene.2020.489357</td>
      <td>Free</td>
      <td>github.com/AlejandroAb/CASCABEL , github.com/frederic-mahe/stampa , github.com/AlejandroAb/CASCABEL-Test/tree/master/mock_analysis , github.com/AlejandroAb/ENA-downloader-tool</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>108</td>
      <td>Natrix: a Snakemake-based workflow for processing, clustering, and taxonomically assigning amplicon sequencing reads</td>
      <td>33198651</td>
      <td>7667751</td>
      <td>10.1186/s12859-020-03852-4</td>
      <td>Free</td>
      <td>github.com/MW55/Natrix</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>109</td>
      <td>Evaluating single-cell cluster stability using the Jaccard similarity index</td>
      <td>33165513</td>
      <td>8352506</td>
      <td>10.1093/bioinformatics/btaa956</td>
      <td>No</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>110</td>
      <td>Reproducibly sampling SARS-CoV-2 genomes across time, geography, and viral diversity</td>
      <td>33500774</td>
      <td>7814287</td>
      <td>10.12688/f1000research.24751.2</td>
      <td>Free</td>
      <td>github.com/nextstrain/augur , github.com/nextstrain/ncov , github.com/caporaso-lab/genome-sampler</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>111</td>
      <td>The Bacteria Genome Pipeline (BAGEP): an automated, scalable workflow for bacteria genomes with Snakemake</td>
      <td>33194387</td>
      <td>7597632</td>
      <td>10.7717/peerj.10121</td>
      <td>Free</td>
      <td>github.com/idolawoye/BAGEP , github.com/idolawoye/BAGEP/blob/master/README</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>112</td>
      <td>MetaLAFFA: a flexible, end-to-end, distributed computing-compatible metagenomic functional annotation pipeline</td>
      <td>33087062</td>
      <td>7579964</td>
      <td>10.1186/s12859-020-03815-9</td>
      <td>Free</td>
      <td>github.com/borenstein-lab/MetaLAFFA</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>113</td>
      <td>Hypercluster: a flexible tool for parallelized unsupervised clustering optimization</td>
      <td>32993491</td>
      <td>7525959</td>
      <td>10.1186/s12859-020-03774-1</td>
      <td>Free</td>
      <td>github.com/ruggleslab/hypercluster , github.com/ruggleslab/hypercluster/tree/master/examples , github.com/ruggleslab/hypercluster/tree/dev/examples , github.com/ruggleslab/hypercluster/</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>114</td>
      <td>DeepRescore: leveraging deep learning to improve peptide identification in immunopeptidomics</td>
      <td>32864883</td>
      <td>7718998</td>
      <td>10.1002/pmic.201900334</td>
      <td>No</td>
      <td></td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>115</td>
      <td>GenomeChronicler: The Personal Genome Project UK Genomic Report Generator Pipeline</td>
      <td>33193602</td>
      <td>7541957</td>
      <td>10.3389/fgene.2020.518644</td>
      <td>Free</td>
      <td>github.com/PGP-UK/GenomeChronicler , github.com/PGP-UK/GenomeChronicler-nf , github.com/PGP-UK/GenomeChronicler-Sarek-nf , github.com/nf-core/sarek</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>116</td>
      <td>Sarek: A portable workflow for whole-genome sequencing analysis of germline and somatic variants</td>
      <td>32269765</td>
      <td>7111497</td>
      <td>10.12688/f1000research.16665.2</td>
      <td>Free</td>
      <td>github.com/nf-core/sarek , github.com/nf-core/configs</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>117</td>
      <td>GeneTEFlow: A Nextflow-based pipeline for analysing gene and transposable elements expression from RNA-Seq data</td>
      <td>32866155</td>
      <td>7458328</td>
      <td>10.1371/journal.pone.0232994</td>
      <td>Free</td>
      <td>github.com/zhongw2/GeneTEFlow</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>118</td>
      <td>gNOMO: a multi-omics pipeline for integrated host and microbiome analysis of non-model organisms</td>
      <td>33575609</td>
      <td>7671378</td>
      <td>10.1093/nargab/lqaa058</td>
      <td>Free</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>119</td>
      <td>Bactopia: a Flexible Pipeline for Complete Analysis of Bacterial Genomes</td>
      <td>32753501</td>
      <td>7406220</td>
      <td>10.1128/mSystems.00190-20</td>
      <td>Free</td>
      <td>github.com/bactopia/bactopia , github.com/bactopia/bactopia/</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>120</td>
      <td>DolphinNext: A graphical user interface for creating, deploying and executing Nextflow pipelines</td>
      <td></td>
      <td>7428130</td>
      <td></td>
      <td>No</td>
      <td></td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>121</td>
      <td>CDSnake: Snakemake pipeline for retrieval of annotated OTUs from paired-end reads using CD-HIT utilities</td>
      <td>32703166</td>
      <td>7379820</td>
      <td>10.1186/s12859-020-03591-6</td>
      <td>Free</td>
      <td>github.com/ydkondratenko/cdsnake</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>122</td>
      <td>lentiMPRA and MPRAflow for high-throughput functional characterization of gene regulatory elements</td>
      <td>32641802</td>
      <td>7550205</td>
      <td>10.1038/s41596-020-0333-5</td>
      <td>No</td>
      <td></td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>123</td>
      <td>ATLAS: a Snakemake workflow for assembly, annotation, and genomic binning of metagenome sequence data</td>
      <td>32571209</td>
      <td>7310028</td>
      <td>10.1186/s12859-020-03585-4</td>
      <td>Free</td>
      <td>github.com/metagenome-atlas/atlas , github.com/metagenome-atlas/supp_data_atlas</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>124</td>
      <td>HemoMIPs—Automated analysis and result reporting pipeline for targeted sequencing data</td>
      <td>32497118</td>
      <td>7297380</td>
      <td>10.1371/journal.pcbi.1007956</td>
      <td>Free</td>
      <td>github.com/kircherlab/hemoMIPs</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>125</td>
      <td>NanoSPC: a scalable, portable, cloud compatible viral nanopore metagenomic data processing pipeline</td>
      <td>32442274</td>
      <td>7319573</td>
      <td>10.1093/nar/gkaa413</td>
      <td>Free</td>
      <td>github.com/rrwick/Porechop</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>126</td>
      <td>DolphinNext: a distributed data processing platform for high throughput genomics</td>
      <td>32306927</td>
      <td>7168977</td>
      <td>10.1186/s12864-020-6714-x</td>
      <td>Free</td>
      <td>github.com/UMMS-Biocore/dolphinnext , github.com/dolphinnext</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>127</td>
      <td>RASflow: an RNA-Seq analysis workflow with Snakemake</td>
      <td>32183729</td>
      <td>7079470</td>
      <td>10.1186/s12859-020-3433-x</td>
      <td>Free</td>
      <td>github.com/zhxiaokang/RASflow , github.com/zhxiaokang/RASflowOperating</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>128</td>
      <td>MasterOfPores: A Workflow for the Analysis of Oxford Nanopore Direct RNA Sequencing Datasets</td>
      <td>32256520</td>
      <td>7089958</td>
      <td>10.3389/fgene.2020.00211</td>
      <td>Free</td>
      <td>github.com/biocorecrg/master_of_pores , github.com/enovoa/EpiNano , github.com/nanoporetech/katuali , github.com/nanoporetech/pipeline-pinfish-analysis , github.com/IOB-Muenster/nanopipe2 , github.com/Psy-Fer/deeplexicon , github.com/lh3/minimap2 , github.com/lbcb-sci/graphmap2 , github.com/wdecoster/NanoPlot , github.com/lpryszcz/bin , github.com/a-slide/NanoCount , github.com/ewels/MultiQC , github.com/jts/nanopolish , github.com/adnaniazi/tailfindr , github.com/nanoporetech/tombo , github.com/biocorecrg/master_of_pores/NanoMod/comparison , github.com/biocorecrg/master_of_pores/tree/master/docker/</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>129</td>
      <td>Can-Seq: a PCR and DNA sequencing strategy for identifying new alleles of known and candidate genes</td>
      <td>32082406</td>
      <td>7017465</td>
      <td>10.1186/s13007-020-0555-0</td>
      <td>Free</td>
      <td>github.com/Carroll-Lab/can_seq</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>130</td>
      <td>CAMITAX: Taxon labels for microbial genomes</td>
      <td>31909794</td>
      <td>6946028</td>
      <td>10.1093/gigascience/giz154</td>
      <td>Free</td>
      <td>github.com/CAMI-challenge/CAMITAX</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>131</td>
      <td>Bioinformatics pipeline using JUDI: Just Do It!</td>
      <td>31882996</td>
      <td>7868055</td>
      <td>10.1093/bioinformatics/btz956</td>
      <td>No</td>
      <td></td>
      <td>Both</td>
    </tr>
    <tr>
      <td>132</td>
      <td>Scalable data analysis in proteomics and metabolomics using BioContainers and workflows engines</td>
      <td>31657527</td>
      <td>7613303</td>
      <td>10.1002/pmic.201900147</td>
      <td>No</td>
      <td></td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>133</td>
      <td>ThermoRawFileParser: modular, scalable and cross-platform RAW file conversion</td>
      <td>31755270</td>
      <td>7116465</td>
      <td>10.1021/acs.jproteome.9b00328</td>
      <td>No</td>
      <td></td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>134</td>
      <td>SNAPPy: A snakemake pipeline for scalable HIV-1 subtyping by phylogenetic pairing</td>
      <td>31768265</td>
      <td>6863187</td>
      <td>10.1093/ve/vez050</td>
      <td>Free</td>
      <td>github.com/PMMAraujo/snappy/releases</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>135</td>
      <td>mPies: a novel metaproteomics tool for the creation of relevant protein databases and automatized protein annotation</td>
      <td>31727118</td>
      <td>6854712</td>
      <td>10.1186/s13062-019-0253-x</td>
      <td>Free</td>
      <td>github.com/johanneswerner/mPies/</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>136</td>
      <td>PyBDA: a command line tool for automated analysis of big biological data sets</td>
      <td>31718539</td>
      <td>6849186</td>
      <td>10.1186/s12859-019-3087-8</td>
      <td>Free</td>
      <td>github.com/cbg-ethz/pybda</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>137</td>
      <td>PhyloMagnet: fast and accurate screening of short-read meta-omics data using gene-centric phylogenetics</td>
      <td>31647547</td>
      <td>7703773</td>
      <td>10.1093/bioinformatics/btz799</td>
      <td>Free</td>
      <td>github.com/maxemil/PhyloMagnet</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>138</td>
      <td>uORF-Tools—Workflow for the determination of translation-regulatory upstream open reading frames</td>
      <td>31513641</td>
      <td>6742470</td>
      <td>10.1371/journal.pone.0222459</td>
      <td>Free</td>
      <td>github.com/Biochemistry1-FFM/uORF-Tools</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>139</td>
      <td>A comparison of automatic cell identification methods for single-cell RNA sequencing data</td>
      <td>31500660</td>
      <td>6734286</td>
      <td>10.1186/s13059-019-1795-z</td>
      <td>Free</td>
      <td>github.com/tabdelaal/scRNAseq_Benchmark , github.com/tabdelaal/scRNAseq_Benchmark/ , github.com/10XGenomics/single-cell-3prime-paper/tree/master/pbmc68k_analysis</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>140</td>
      <td>Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology</td>
      <td>31510660</td>
      <td>6612828</td>
      <td>10.1093/bioinformatics/btz363</td>
      <td>Free</td>
      <td>github.com/grst/immune_deconvolution_benchmark , github.com/grst/immunedeconv</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>141</td>
      <td>Eye in a Disk: eyeIntegration Human Pan-Eye and Body Transcriptome Database Version 1.0</td>
      <td>31343654</td>
      <td>6660187</td>
      <td>10.1167/iovs.19-27106</td>
      <td>Free</td>
      <td>github.com/davemcg/eyeIntegration_app/ , github.com/davemcg/EiaD_build , github.com/davemcg/eyeIntegration_v1_app_manuscript , github.com/davemcg/eyeIntegration_app</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>142</td>
      <td>Pipeliner: A Nextflow-Based Framework for the Definition of Sequencing Data Processing Pipelines</td>
      <td>31316552</td>
      <td>6609566</td>
      <td>10.3389/fgene.2019.00614</td>
      <td>Free</td>
      <td>github.com/montilab/pipeliner</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>143</td>
      <td>ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data</td>
      <td>31088905</td>
      <td>6643886</td>
      <td>10.1534/g3.119.400185</td>
      <td>Free</td>
      <td>github.com/csoneson/ARMOR , github.com/csoneson/ARMOR/wiki , github.com/csoneson/ARMOR/blob/chiron_realdataworkflow/E-MTAB-7029/download_files , github.com/csoneson/ARMOR/blob/chiron_realdataworkflow/config_E-MTAB-7029 , github.com/csoneson/ARMOR/tree/chiron_realdataworkflow , github.com/csoneson/ARMOR/blob/chiron_realdataworkflow/E-MTAB-7029/run_iSEE</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>144</td>
      <td>Sunbeam: an extensible pipeline for analyzing metagenomic sequencing experiments</td>
      <td>30902113</td>
      <td>6429786</td>
      <td>10.1186/s40168-019-0658-x</td>
      <td>Free</td>
      <td>github.com/sunbeam-labs/sunbeam , github.com/sunbeam-labs/sbx_template , github.com/eclarke/komplexity , github.com/louiejtaylor/sbx_lewis2015 , github.com/louiejtaylor/sbx_bahram2018 , github.com/louiejtaylor/sbx_mccann2018 , github.com/junglee0713/sbx_shallowshotgun_pilot , github.com/sunbeam-labs</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>145</td>
      <td>Integrating Bacterial ChIP-seq and RNA-seq data with SnakeChunks</td>
      <td>30786165</td>
      <td>7302399</td>
      <td>10.1002/cpbi.72</td>
      <td>No</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>146</td>
      <td>Scalable Workflows and Reproducible Data Analysis for Genomics</td>
      <td>31278683</td>
      <td>7613310</td>
      <td>10.1007/978-1-4939-9074-0_24</td>
      <td>Free</td>
      <td>github.com/EvolutionaryGenomics/scalability-reproducibility-chapter</td>
      <td>Both</td>
    </tr>
    <tr>
      <td>147</td>
      <td>Developing reproducible bioinformatics analysis workflows for heterogeneous computing environments to support African genomics</td>
      <td>30486782</td>
      <td>6264621</td>
      <td>10.1186/s12859-018-2446-1</td>
      <td>Free</td>
      <td>github.com/NERSC/shifter , github.com/h3abionet/h3agatkand , github.com/h3abionet/h3agatk , github.com/h3abionet/h3abionet16S , github.com/h3abionet/h3agwas , github.com/h3abionet/chipimputation , github.com/h3abionet/h3abionet16S/tree/master/workflows-cwl , github.com/h3abionet/h3agwas/ , github.com/wtsi-hgi/gatk-cwl-generator/ , github.com/EBI-Metagenomics/ebi-metagenomics-cwl , github.com/h3abionet/h3abionet16S/tree/master/workflows-nxf , github.com/h3abionetOperating</td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>148</td>
      <td>Pergola: Boosting Visualization and Analysis of Longitudinal Data by Unlocking Genomic Analysis Tools</td>
      <td>30419504</td>
      <td>6231116</td>
      <td>10.1016/j.isci.2018.10.023</td>
      <td>Free</td>
      <td></td>
      <td>Nextflow</td>
    </tr>
    <tr>
      <td>149</td>
      <td>ProteomeGenerator: A Framework for Comprehensive Proteomics Based on de Novo Transcriptome Assembly and High-Accuracy Peptide Mass Spectral Matching</td>
      <td>30295032</td>
      <td>6727203</td>
      <td>10.1021/acs.jproteome.8b00295</td>
      <td>No</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>150</td>
      <td>A Bayesian model for single cell transcript expression analysis on MERFISH data</td>
      <td>30875429</td>
      <td>6419903</td>
      <td>10.1093/bioinformatics/bty718</td>
      <td>No</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>151</td>
      <td>ATAC2GRN: optimized ATAC-seq and DNase1-seq pipelines for rapid and accurate genome regulatory network inference</td>
      <td>30064353</td>
      <td>6069842</td>
      <td>10.1186/s12864-018-4943-z</td>
      <td>Free</td>
      <td>github.com/ChioriniLab , github.com/lh3/seqtk</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>152</td>
      <td>Sequanix: a dynamic graphical interface for Snakemake workflows</td>
      <td>29361152</td>
      <td>5972652</td>
      <td>10.1093/bioinformatics/bty034</td>
      <td>Free</td>
      <td>github.com/sequana/sequana</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>153</td>
      <td>TaxMapper: an analysis tool, reference database and workflow for metatranscriptome analysis of eukaryotic microorganisms</td>
      <td>29037173</td>
      <td>5644092</td>
      <td>10.1186/s12864-017-4168-6</td>
      <td>Free</td>
      <td></td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>154</td>
      <td>MetaMeta: integrating metagenome analysis tools to improve taxonomic profiling</td>
      <td>28807044</td>
      <td>5557516</td>
      <td>10.1186/s40168-017-0318-y</td>
      <td>Free</td>
      <td>github.com/pirovc/metameta , github.com/bioboxes/rfc/tree/master/data-format , github.com/pirovc/metametaOperating , github.com/pirovc/metametamerge</td>
      <td>Snakemake</td>
    </tr>
    <tr>
      <td>155</td>
      <td>An automated workflow for parallel processing of large multiview SPIM recordings</td>
      <td>26628585</td>
      <td>4896369</td>
      <td>10.1093/bioinformatics/btv706</td>
      <td>Free</td>
      <td>github.com/mpicbg-scicomp/snakemake-workflows , github.com/fiji/SPIM_Registration</td>
      <td>Snakemake</td>
    </tr>
  </tbody>
</table>