{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "d5a8faf0",
   "metadata": {},
   "source": [
    "# Extraction et Création des fichiers txt contenant les articles\n",
    "Clemence SEBE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "0bad6320",
   "metadata": {},
   "outputs": [],
   "source": [
    "import xml.etree.ElementTree as et \n",
    "import pandas as pd\n",
    "import re\n",
    "import os"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a6ab3a16",
   "metadata": {},
   "source": [
    "# Extraction des Articles de PMC du fichier xml"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "2c0245a1",
   "metadata": {},
   "outputs": [],
   "source": [
    "pmcAll = et.parse('9_10_2022/q3_9_10_2022.xml')\n",
    "pmc = pmcAll.getroot()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8f020c3f",
   "metadata": {},
   "source": [
    "Fonction intermédiare permettant d'extraire certaines informations et le texte du XML : "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "df308b61",
   "metadata": {},
   "outputs": [],
   "source": [
    "def extractText(part, sup=False):\n",
    "    txt = ''\n",
    "    if part.tag != 'fig' and part.tag != 'table' and part.tag != 'table-wrap':  \n",
    "        \n",
    "        if part.tag == 'list-item':\n",
    "            for subPart in part:\n",
    "                if subPart.tag != 'label':\n",
    "                    txt += '** ' + extractText(subPart) + '\\n'\n",
    "        else:\n",
    "            if sup:\n",
    "                txt += ' ['\n",
    "                \n",
    "            if part.text != None:\n",
    "                if len(part) == 0:\n",
    "                    txt += part.text\n",
    "                else: \n",
    "                    txt += (part.text).strip() + \" \"\n",
    "\n",
    "            for subPart in part:\n",
    "                if subPart.tag =='xref':\n",
    "                    txt = txt.strip() \n",
    "                if subPart.tag == 'ext-link':\n",
    "                    txt = txt.strip() + \" \"\n",
    "                    \n",
    "                if subPart.tag == 'sup':\n",
    "                    txt = txt.strip()\n",
    "                    txt += extractText(subPart, True)\n",
    "                else:\n",
    "                    txt += extractText(subPart)\n",
    "\n",
    "            if sup:\n",
    "                txt += ']'\n",
    "            if part.tail != None:\n",
    "                if len(part.tail.strip()) != 0:\n",
    "                    txt += part.tail\n",
    "    \n",
    "    return txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "55a3167f",
   "metadata": {},
   "outputs": [],
   "source": [
    "def extracTitle(body, tab, idx):\n",
    "    section = body.findall('sec')\n",
    "    for sec in section:\n",
    "        title = sec.find('title')\n",
    "        if title != None:\n",
    "            txt = extractText(title)\n",
    "               \n",
    "            new = tab\n",
    "            new.append([idx,txt])\n",
    "            extracTitle(sec, new , idx+1)\n",
    "        else:\n",
    "            extracTitle(sec,tab, idx+1)\n",
    "    return tab"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "06b060b7",
   "metadata": {},
   "outputs": [],
   "source": [
    "def whichSection(titres, wordsAccepted):\n",
    "    accepted = []\n",
    "    for i in range (len(titres)) :\n",
    "        word = titres[i][-1].lower()\n",
    "        for wA in wordsAccepted:\n",
    "            if word.find(wA) != -1:\n",
    "                ok = True\n",
    "                if titres[i][0] == 0:\n",
    "                    if not titres[i][-1] in accepted:\n",
    "                        accepted.append(titres[i][-1])\n",
    "                else:\n",
    "                    for j in range (i,-1,-1):\n",
    "                        if titres[j][0] == 0:\n",
    "                            if not titres[j][-1] in accepted:\n",
    "                                accepted.append(titres[j][-1])\n",
    "                            break\n",
    "            \n",
    "    return accepted"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "97549f80",
   "metadata": {},
   "outputs": [],
   "source": [
    "def extractTextSection(file, section):\n",
    "    for part in section:\n",
    "        if part.tag == 'fn-group':\n",
    "            extractTextSection(file,part) \n",
    "        if part.tag == 'title':\n",
    "            txt = '\\n\\n\\t' + extractText(part) + '\\n\\n'\n",
    "            file.write(txt)\n",
    "        if part.tag == 'p' or part.tag == 'fn':\n",
    "            txt = extractText(part) + '\\n'\n",
    "            file.write(txt)\n",
    "        if part.tag == 'list':\n",
    "            txt = extractText(part) \n",
    "            file.write(txt)\n",
    "        if part.tag == 'notes':\n",
    "            extractTextSection(file,part)\n",
    "        if part.tag == 'sec':\n",
    "            extractTextSection(file, part)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "05c26636",
   "metadata": {},
   "source": [
    "\"Main\" créant les différents fichier txt composant par la suite le corpus :"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "05c87e32",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1/110\n",
      "A bioinformatic-assisted workflow for genome-wide identification of ncRNAs\n",
      "~~~~~~~~~~~~~~~~~\n",
      "2/110\n",
      "BioUML—towards a universal research platform\n",
      "~~~~~~~~~~~~~~~~~\n",
      "3/110\n",
      "Detection and Validation of Circular DNA Fragments Using Nanopore Sequencing\n",
      "~~~~~~~~~~~~~~~~~\n",
      "4/110\n",
      "GenErode: a bioinformatics pipeline to investigate genome erosion in endangered and extinct species\n",
      "~~~~~~~~~~~~~~~~~\n",
      "5/110\n",
      "nf-gwas-pipeline: A Nextflow Genome-Wide Association Study Pipeline\n",
      "~~~~~~~~~~~~~~~~~\n",
      "6/110\n",
      "~~~~~~~~~~~~~~~~~\n",
      "7/110\n",
      "NanoRTax, a real-time pipeline for taxonomic and diversity analysis of nanopore 16S rRNA amplicon sequencing data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "8/110\n",
      "MetaWorks: A flexible, scalable bioinformatic pipeline for high-throughput multi-marker biodiversity assessments\n",
      "~~~~~~~~~~~~~~~~~\n",
      "9/110\n",
      "DetectIS: a pipeline to rapidly detect exogenous DNA integration sites using DNA or RNA paired-end sequencing data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "10/110\n",
      "TMBur: a distributable tumor mutation burden approach for whole genome sequencing\n",
      "~~~~~~~~~~~~~~~~~\n",
      "11/110\n",
      "Scalable Workflows and Reproducible Data Analysis for Genomics\n",
      "~~~~~~~~~~~~~~~~~\n",
      "12/110\n",
      "Tourmaline: A containerized workflow for rapid and iterable amplicon sequence analysis using QIIME 2 and Snakemake\n",
      "~~~~~~~~~~~~~~~~~\n",
      "13/110\n",
      "epiGBS2: Improvements and evaluation of highly multiplexed, epiGBS‐based reduced representation bisulfite sequencing\n",
      "~~~~~~~~~~~~~~~~~\n",
      "14/110\n",
      "iCOMIC: a graphical interface-driven bioinformatics pipeline for analyzing cancer omics data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "15/110\n",
      "Spacemake: processing and analysis of large-scale spatial transcriptomics data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "16/110\n",
      "Implementing the reuse of public DIA proteomics datasets: from the PRIDE database to Expression Atlas\n",
      "~~~~~~~~~~~~~~~~~\n",
      "17/110\n",
      "Exodus: sequencing-based pipeline for quantification of pooled variants\n",
      "~~~~~~~~~~~~~~~~~\n",
      "18/110\n",
      "GEMmaker: process massive RNA-seq datasets on heterogeneous computational infrastructure\n",
      "~~~~~~~~~~~~~~~~~\n",
      "19/110\n",
      "Findzx: an automated pipeline for detecting and visualising sex chromosomes using whole-genome sequencing data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "20/110\n",
      "A bioinformatic pipeline for simulating viral integration data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "21/110\n",
      "MEDUSA: A Pipeline for Sensitive Taxonomic Classification and Flexible Functional Annotation of Metagenomic Shotgun Sequences\n",
      "~~~~~~~~~~~~~~~~~\n",
      "22/110\n",
      "Enhanced protein isoform characterization through long-read proteogenomics\n",
      "~~~~~~~~~~~~~~~~~\n",
      "23/110\n",
      "Topological approximate Bayesian computation for parameter inference of an angiogenesis model\n",
      "~~~~~~~~~~~~~~~~~\n",
      "24/110\n",
      "Phylodynamic Inference of Bacterial Outbreak Parameters Using Nanopore Sequencing\n",
      "~~~~~~~~~~~~~~~~~\n",
      "25/110\n",
      "riboviz 2: a flexible and robust ribosome profiling data analysis and visualization workflow\n",
      "~~~~~~~~~~~~~~~~~\n",
      "26/110\n",
      "SWAAT Bioinformatics Workflow for Protein Structure-Based Annotation of ADME Gene Variants\n",
      "~~~~~~~~~~~~~~~~~\n",
      "27/110\n",
      "Side-by-Side Comparison of Post-Entry Quarantine and High Throughput Sequencing Methods for Virus and Viroid Diagnosis\n",
      "~~~~~~~~~~~~~~~~~\n",
      "28/110\n",
      "nf-core/mag: a best-practice pipeline for metagenome hybrid assembly and binning\n",
      "~~~~~~~~~~~~~~~~~\n",
      "29/110\n",
      "DENTIST—using long reads for closing assembly gaps at high accuracy\n",
      "~~~~~~~~~~~~~~~~~\n",
      "30/110\n",
      "nf-rnaSeqCount: A Nextflow pipeline for obtaining raw read counts from RNA-seq data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "31/110\n",
      "Identifying cancer pathway dysregulations using differential causal effects\n",
      "~~~~~~~~~~~~~~~~~\n",
      "32/110\n",
      "Clones and Clusters of Antimicrobial-Resistant Klebsiella From Southwestern Nigeria\n",
      "~~~~~~~~~~~~~~~~~\n",
      "33/110\n",
      "snpQT: flexible, reproducible, and comprehensive quality control and imputation of genomic data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "34/110\n",
      "Comprehensive Analysis of Large-Scale Transcriptomes from Multiple Cancer Types\n",
      "~~~~~~~~~~~~~~~~~\n",
      "35/110\n",
      "nextNEOpi: a comprehensive pipeline for computational neoantigen prediction\n",
      "~~~~~~~~~~~~~~~~~\n",
      "36/110\n",
      "Design considerations for workflow management systems use in production genomics research and the clinic\n",
      "~~~~~~~~~~~~~~~~~\n",
      "37/110\n",
      "ORPER: A Workflow for Constrained SSU rRNA Phylogenies\n",
      "~~~~~~~~~~~~~~~~~\n",
      "38/110\n",
      "FA-nf: A Functional Annotation Pipeline for Proteins from Non-Model Organisms Implemented in Nextflow\n",
      "~~~~~~~~~~~~~~~~~\n",
      "39/110\n",
      "Characterisation of tumour microenvironment remodelling following oncogene inhibition in preclinical studies with imaging mass cytometry\n",
      "~~~~~~~~~~~~~~~~~\n",
      "40/110\n",
      "MuWU: Mutant-seq library analysis and annotation\n",
      "~~~~~~~~~~~~~~~~~\n",
      "41/110\n",
      "Genetic demultiplexing of pooled single-cell RNA-sequencing samples in cancer facilitates effective experimental design\n",
      "~~~~~~~~~~~~~~~~~\n",
      "42/110\n",
      "VGEA: an RNA viral assembly toolkit\n",
      "~~~~~~~~~~~~~~~~~\n",
      "43/110\n",
      "ExOrthist: a tool to infer exon orthologies at any evolutionary distance\n",
      "~~~~~~~~~~~~~~~~~\n",
      "44/110\n",
      "Large-scale quality assessment of prokaryotic genomes with metashot/prok-quality\n",
      "~~~~~~~~~~~~~~~~~\n",
      "45/110\n",
      "nf-LO: A Scalable, Containerized Workflow for Genome-to-Genome Lift Over\n",
      "~~~~~~~~~~~~~~~~~\n",
      "46/110\n",
      "ppx: Programmatic access to proteomics data repositories\n",
      "~~~~~~~~~~~~~~~~~\n",
      "47/110\n",
      "poreCov-An Easy to Use, Fast, and Robust Workflow for SARS-CoV-2 Genome Reconstruction via Nanopore Sequencing\n",
      "~~~~~~~~~~~~~~~~~\n",
      "48/110\n",
      "BIGwas: Single-command quality control and association testing for multi-cohort and biobank-scale GWAS/PheWAS data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "49/110\n",
      "MicroPIPE: validating an end-to-end workflow for high-quality complete bacterial genome construction\n",
      "~~~~~~~~~~~~~~~~~\n",
      "50/110\n",
      "Systematic benchmarking of tools for CpG methylation detection from nanopore sequencing\n",
      "~~~~~~~~~~~~~~~~~\n",
      "51/110\n",
      "The Dockstore: enhancing a community platform for sharing reproducible and accessible computational protocols\n",
      "~~~~~~~~~~~~~~~~~\n",
      "52/110\n",
      "Species-Specific Quality Control, Assembly and Contamination Detection in Microbial Isolate Sequences with AQUAMIS\n",
      "~~~~~~~~~~~~~~~~~\n",
      "53/110\n",
      "Sustainable data analysis with Snakemake\n",
      "~~~~~~~~~~~~~~~~~\n",
      "54/110\n",
      "Testcrosses are an efficient strategy for identifying cis-regulatory variation: Bayesian analysis of allele-specific expression (BayesASE)\n",
      "~~~~~~~~~~~~~~~~~\n",
      "55/110\n",
      "WASP: a versatile, web-accessible single cell RNA-Seq processing platform\n",
      "~~~~~~~~~~~~~~~~~\n",
      "56/110\n",
      "Reproducible, portable, and efficient ancient genome reconstruction with nf-core/eager\n",
      "~~~~~~~~~~~~~~~~~\n",
      "57/110\n",
      "~~~~~~~~~~~~~~~~~\n",
      "58/110\n",
      "Using prototyping to choose a bioinformatics workflow management system\n",
      "~~~~~~~~~~~~~~~~~\n",
      "59/110\n",
      "Metagenomics workflow for hybrid assembly, differential coverage binning, metatranscriptomics and pathway analysis (MUFFIN)\n",
      "~~~~~~~~~~~~~~~~~\n",
      "60/110\n",
      "lncEvo: automated identification and conservation study of long noncoding RNAs\n",
      "~~~~~~~~~~~~~~~~~\n",
      "61/110\n",
      "RNAflow: An Effective and Simple RNA-Seq Differential Gene Expression Pipeline Using Nextflow\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "~~~~~~~~~~~~~~~~~\n",
      "62/110\n",
      "Dadasnake, a Snakemake implementation of DADA2 to process amplicon sequencing data for microbial ecology\n",
      "~~~~~~~~~~~~~~~~~\n",
      "63/110\n",
      "Accurate and Efficient KIR Gene and Haplotype Inference From Genome Sequencing Reads With Novel K-mer Signatures\n",
      "~~~~~~~~~~~~~~~~~\n",
      "64/110\n",
      "Cascabel: A Scalable and Versatile Amplicon Sequence Data Analysis Pipeline Delivering Reproducible and Documented Results\n",
      "~~~~~~~~~~~~~~~~~\n",
      "65/110\n",
      "Natrix: a Snakemake-based workflow for processing, clustering, and taxonomically assigning amplicon sequencing reads\n",
      "~~~~~~~~~~~~~~~~~\n",
      "66/110\n",
      "~~~~~~~~~~~~~~~~~\n",
      "67/110\n",
      "Reproducibly sampling SARS-CoV-2 genomes across time, geography, and viral diversity\n",
      "~~~~~~~~~~~~~~~~~\n",
      "68/110\n",
      "The Bacteria Genome Pipeline (BAGEP): an automated, scalable workflow for bacteria genomes with Snakemake\n",
      "~~~~~~~~~~~~~~~~~\n",
      "69/110\n",
      "MetaLAFFA: a flexible, end-to-end, distributed computing-compatible metagenomic functional annotation pipeline\n",
      "~~~~~~~~~~~~~~~~~\n",
      "70/110\n",
      "Hypercluster: a flexible tool for parallelized unsupervised clustering optimization\n",
      "~~~~~~~~~~~~~~~~~\n",
      "71/110\n",
      "~~~~~~~~~~~~~~~~~\n",
      "72/110\n",
      "GenomeChronicler: The Personal Genome Project UK Genomic Report Generator Pipeline\n",
      "~~~~~~~~~~~~~~~~~\n",
      "73/110\n",
      "Sarek: A portable workflow for whole-genome sequencing analysis of germline and somatic variants\n",
      "~~~~~~~~~~~~~~~~~\n",
      "74/110\n",
      "GeneTEFlow: A Nextflow-based pipeline for analysing gene and transposable elements expression from RNA-Seq data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "75/110\n",
      "gNOMO: a multi-omics pipeline for integrated host and microbiome analysis of non-model organisms\n",
      "~~~~~~~~~~~~~~~~~\n",
      "76/110\n",
      "Bactopia: a Flexible Pipeline for Complete Analysis of Bacterial Genomes\n",
      "~~~~~~~~~~~~~~~~~\n",
      "77/110\n",
      "~~~~~~~~~~~~~~~~~\n",
      "78/110\n",
      "CDSnake: Snakemake pipeline for retrieval of annotated OTUs from paired-end reads using CD-HIT utilities\n",
      "~~~~~~~~~~~~~~~~~\n",
      "79/110\n",
      "~~~~~~~~~~~~~~~~~\n",
      "80/110\n",
      "ATLAS: a Snakemake workflow for assembly, annotation, and genomic binning of metagenome sequence data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "81/110\n",
      "HemoMIPs—Automated analysis and result reporting pipeline for targeted sequencing data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "82/110\n",
      "NanoSPC: a scalable, portable, cloud compatible viral nanopore metagenomic data processing pipeline\n",
      "~~~~~~~~~~~~~~~~~\n",
      "83/110\n",
      "DolphinNext: a distributed data processing platform for high throughput genomics\n",
      "~~~~~~~~~~~~~~~~~\n",
      "84/110\n",
      "RASflow: an RNA-Seq analysis workflow with Snakemake\n",
      "~~~~~~~~~~~~~~~~~\n",
      "85/110\n",
      "MasterOfPores: A Workflow for the Analysis of Oxford Nanopore Direct RNA Sequencing Datasets\n",
      "~~~~~~~~~~~~~~~~~\n",
      "86/110\n",
      "Can-Seq: a PCR and DNA sequencing strategy for identifying new alleles of known and candidate genes\n",
      "~~~~~~~~~~~~~~~~~\n",
      "87/110\n",
      "CAMITAX: Taxon labels for microbial genomes\n",
      "~~~~~~~~~~~~~~~~~\n",
      "88/110\n",
      "~~~~~~~~~~~~~~~~~\n",
      "89/110\n",
      "~~~~~~~~~~~~~~~~~\n",
      "90/110\n",
      "SNAPPy: A snakemake pipeline for scalable HIV-1 subtyping by phylogenetic pairing\n",
      "~~~~~~~~~~~~~~~~~\n",
      "91/110\n",
      "mPies: a novel metaproteomics tool for the creation of relevant protein databases and automatized protein annotation\n",
      "~~~~~~~~~~~~~~~~~\n",
      "92/110\n",
      "PyBDA: a command line tool for automated analysis of big biological data sets\n",
      "~~~~~~~~~~~~~~~~~\n",
      "93/110\n",
      "PhyloMagnet: fast and accurate screening of short-read meta-omics data using gene-centric phylogenetics\n",
      "~~~~~~~~~~~~~~~~~\n",
      "94/110\n",
      "uORF-Tools—Workflow for the determination of translation-regulatory upstream open reading frames\n",
      "~~~~~~~~~~~~~~~~~\n",
      "95/110\n",
      "A comparison of automatic cell identification methods for single-cell RNA sequencing data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "96/110\n",
      "Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology\n",
      "~~~~~~~~~~~~~~~~~\n",
      "97/110\n",
      "Eye in a Disk: eyeIntegration Human Pan-Eye and Body Transcriptome Database Version 1.0\n",
      "~~~~~~~~~~~~~~~~~\n",
      "98/110\n",
      "Pipeliner: A Nextflow-Based Framework for the Definition of Sequencing Data Processing Pipelines\n",
      "~~~~~~~~~~~~~~~~~\n",
      "99/110\n",
      "ARMOR: An Automated Reproducible MOdular Workflow for Preprocessing and Differential Analysis of RNA-seq Data\n",
      "~~~~~~~~~~~~~~~~~\n",
      "100/110\n",
      "Sunbeam: an extensible pipeline for analyzing metagenomic sequencing experiments\n",
      "~~~~~~~~~~~~~~~~~\n",
      "101/110\n",
      "~~~~~~~~~~~~~~~~~\n",
      "102/110\n",
      "Developing reproducible bioinformatics analysis workflows for heterogeneous computing environments to support African genomics\n",
      "~~~~~~~~~~~~~~~~~\n",
      "103/110\n",
      "Pergola: Boosting Visualization and Analysis of Longitudinal Data by Unlocking Genomic Analysis Tools\n",
      "~~~~~~~~~~~~~~~~~\n",
      "104/110\n",
      "~~~~~~~~~~~~~~~~~\n",
      "105/110\n",
      "~~~~~~~~~~~~~~~~~\n",
      "106/110\n",
      "ATAC2GRN: optimized ATAC-seq and DNase1-seq pipelines for rapid and accurate genome regulatory network inference\n",
      "~~~~~~~~~~~~~~~~~\n",
      "107/110\n",
      "Sequanix: a dynamic graphical interface for Snakemake workflows\n",
      "~~~~~~~~~~~~~~~~~\n",
      "108/110\n",
      "TaxMapper: an analysis tool, reference database and workflow for metatranscriptome analysis of eukaryotic microorganisms\n",
      "~~~~~~~~~~~~~~~~~\n",
      "109/110\n",
      "MetaMeta: integrating metagenome analysis tools to improve taxonomic profiling\n",
      "~~~~~~~~~~~~~~~~~\n",
      "110/110\n",
      "An automated workflow for parallel processing of large multiview SPIM recordings\n",
      "~~~~~~~~~~~~~~~~~\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n",
      "/tmp/ipykernel_6090/3205428600.py:150: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid,\n"
     ]
    }
   ],
   "source": [
    "wordsAccepted = ['implementation', 'material', 'method', 'operation', 'pipeline', 'workflow', 'tool', '\\u2003']\n",
    "access = 0\n",
    "nonAccess = 0\n",
    "dicoArticle = {}\n",
    "idx = 1\n",
    "\n",
    "#a optimiser\n",
    "try:\n",
    "    os.mkdir(\"article\")\n",
    "    os.mkdir(\"article/all\")\n",
    "except:\n",
    "    None\n",
    "\n",
    "\n",
    "titre = []\n",
    "titleSection = open(\"sectionTitle.txt\", 'w')\n",
    "\n",
    "paternGit = '(https:\\/\\/)?(www\\.)?github\\.com\\/(\\w|\\/|-|_)*'\n",
    "\n",
    "df = pd.DataFrame(columns=['id','titre', 'pmid', 'pmc', 'doi', 'XMLaccess', 'git', 'language'])                   \n",
    "\n",
    "for article in pmc:\n",
    "    print(str(idx) + \"/\" + str(len(pmc)))\n",
    "    front = article.find('front')\n",
    "    body = article.find('body')\n",
    "    \n",
    "    #extract gloabal information on the article\n",
    "    dicoSubArticle = {}\n",
    "    dicoSubArticle[idx] = {}\n",
    "    articleMeta = front.find('article-meta')\n",
    "    temp = {}\n",
    "    for idA in articleMeta.findall('article-id'):\n",
    "        if not idA.attrib['pub-id-type'] in temp:\n",
    "            temp.update({idA.attrib['pub-id-type'] : idA.text})\n",
    "        else:\n",
    "            print('NORMALLY NO - NOT TWO ID DIFFERENT')\n",
    "            \n",
    "    for k in temp:\n",
    "        dicoSubArticle[idx][k] = temp[k]\n",
    "    \n",
    "    title = articleMeta.find('title-group')\n",
    "    txtTitle = []\n",
    "    for child in title:\n",
    "        if child.tag.find('title') != -1:\n",
    "            txt = extractText(child)\n",
    "            txtTitle.append(txt)\n",
    "    dicoSubArticle[idx]['title'] = txtTitle\n",
    "    \n",
    "    #  abstract\n",
    "    string = 'article/all/' + str(idx) + '.all.txt'\n",
    "    fileAll = open(string,'w')\n",
    "    fileAll.write(dicoSubArticle[idx]['title'][0] + '\\n\\n')\n",
    "        \n",
    "    abstract = articleMeta.findall('abstract')\n",
    "    for i in range (len(abstract)):\n",
    "        if i > 0:\n",
    "            fileAll.write(\"\\n\" + \"Abstract \" + str(i) + ' :\\n')\n",
    "        else:\n",
    "            fileAll.write(\"Abstract :\\n\")\n",
    "        extractTextSection(fileAll, abstract[i])\n",
    "    fileAll.write(\"\\n\" + \"-\"*100 + \"\\n\")\n",
    "\n",
    "    \n",
    "    if body == None:\n",
    "        nonAccess += 1\n",
    "        dicoSubArticle[idx]['XMLaccess'] = 'No'\n",
    "        dicoSubArticle[idx]['git'] = ''\n",
    "        fileAll.close()\n",
    "        stringGit = ''\n",
    "        tabGit = []\n",
    "    else:\n",
    "        access += 1\n",
    "        dicoSubArticle[idx]['XMLaccess'] = 'Free'\n",
    "        \n",
    "        #extract the text - only some sections       \n",
    "        titres = extracTitle(body,[], 0)\n",
    "        string = str(idx) + \" \"+ dicoSubArticle[idx]['title'][0]\n",
    "        titleSection.write(string)\n",
    "        titleSection.write('\\n')\n",
    "        for t in titres:\n",
    "            titleSection.write('\\t'*t[0])\n",
    "            titleSection.write(t[1])\n",
    "            titleSection.write('\\n')\n",
    "        titleSection.write('\\n\\n')\n",
    "        \n",
    "        nameSection = whichSection(titres, wordsAccepted)\n",
    "        try:\n",
    "            string = 'article/' + str(idx) + \"_PMID\" + dicoSubArticle[idx]['pmid'] + '.txt'\n",
    "        except:\n",
    "            string = 'article/' + str(idx) + \"_PMC\" + dicoSubArticle[idx]['pmc'] + '.txt'\n",
    "        file = open(string,'w')\n",
    "        file.write(dicoSubArticle[idx]['title'][0] + '\\n')\n",
    "        \n",
    "        print(dicoSubArticle[idx]['title'][0])\n",
    "        section = body.findall('sec')\n",
    "        for sec in section:\n",
    "            title = sec.find('title')\n",
    "            if title != None:\n",
    "                txt = extractText(title)\n",
    "                if txt in nameSection:\n",
    "                    extractTextSection(file,sec)\n",
    "        file.close()\n",
    "        \n",
    "        #extract all the article\n",
    "        for sec in section:\n",
    "            extractTextSection(fileAll,sec)\n",
    "        \n",
    "        #and the section Data Availability if the article have one\n",
    "        fileAll.write(\"\\n\" + \"-\"*100 + \"\\n\")\n",
    "        back = article.find('back')\n",
    "        extractTextSection(fileAll, back)               \n",
    "            \n",
    "        fileAll.close() \n",
    "        \n",
    "        #extract github @ \n",
    "        string = 'article/all/' + str(idx) + '.all.txt'\n",
    "        fileR = open(string,'r')\n",
    "        txt = fileR.read()\n",
    "        tabGit = []\n",
    "        stringGit = ''\n",
    "        for match in re.finditer(paternGit, txt):\n",
    "            git = txt[match.span()[0]: match.span()[1]]\n",
    "            \n",
    "            if git[0] == 'h':\n",
    "                git = git[8:]\n",
    "            if git[0] == 'w':\n",
    "                git = git[4:]\n",
    "            if not git in tabGit:\n",
    "                tabGit.append(git)\n",
    "                stringGit += git + ' , '\n",
    "        stringGit = stringGit[:-(len(' , '))]\n",
    "        dicoSubArticle[idx]['git'] = stringGit\n",
    "        fileR.close()\n",
    "        \n",
    "            \n",
    "\n",
    "    try:\n",
    "        pmidid = dicoSubArticle[idx]['pmid'] \n",
    "    except :\n",
    "        pmidid = ''\n",
    "    try:\n",
    "        pmcid = dicoSubArticle[idx]['pmc'] \n",
    "    except :\n",
    "        pmcid = ''\n",
    "    try:\n",
    "        doiid = dicoSubArticle[idx]['doi'] \n",
    "    except :\n",
    "        doiid = ''\n",
    "        \n",
    "    df = df.append({'id':idx, 'titre':dicoSubArticle[idx]['title'][0], 'pmid':pmidid, 'pmc':pmcid, \n",
    "                'doi':doiid, 'XMLaccess':dicoSubArticle[idx]['XMLaccess'], 'git':dicoSubArticle[idx]['git']}, ignore_index=True)\n",
    "    \n",
    "    print(\"~~~~~~~~~~~~~~~~~\")\n",
    "    \n",
    "    #print(dicoSubArticle)\n",
    "    dicoArticle.update(dicoSubArticle)\n",
    "    idx += 1\n",
    "    \n",
    "#df.to_csv( language + \"_corpus.csv\", index=False)\n",
    "#df.to_html(language + '_corpus.html',justify='center',index=False)\n",
    "titleSection.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "03e3b3db",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "On 110 articles, 99 open access Articles and 11 no XML open acess Articles\n"
     ]
    }
   ],
   "source": [
    "print(\"On {} articles, {} open access Articles and {} no XML open acess Articles\".format(len(pmc),access, nonAccess))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "58b5056d",
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "# A extraire : \n",
    "# implementation - Materials and Methods - Methods - Operation - Pipeline - Workflow - Library?? - Tool(s)\n",
    "#Peut etre :\n",
    "# Availability and requirements (*5)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a930b660",
   "metadata": {},
   "source": [
    "# Deplacement des articles dans le bon dossier "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "a8869ff9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Nombre d'articles decrivant des wf Nextflow : 48\n",
      "Nombre d'articles decrivant des wf Snakemake : 49\n",
      "Nombre d'articles possédant à la fois Nextflow et Snakemake : 3\n"
     ]
    }
   ],
   "source": [
    "path = 'article/all/'\n",
    "listArticle = os.listdir(path)\n",
    "\n",
    "nbNextflow = 0\n",
    "nbSnakemake = 0\n",
    "nbBoth = 0\n",
    "nonAccess = 0\n",
    "\n",
    "\"\"\"try:\n",
    "    os.mkdir('nextflow')\n",
    "    os.mkdir('nextflow/all')\n",
    "    os.mkdir('snakemake')\n",
    "    os.mkdir('snakemake/all')\n",
    "except:\n",
    "    None\"\"\"\n",
    "    \n",
    "\n",
    "for article in listArticle:\n",
    "    a = open(path + article, 'r')\n",
    "    idxDf = int(article.split('.')[0]) -1\n",
    "    txt = a.read()\n",
    "    idx = txt.find('-'*100)\n",
    "    abstract = txt[:idx].lower()\n",
    "\n",
    "    n = abstract.find('nextflow')\n",
    "    s = abstract.find('snakemake')\n",
    "    \n",
    "    if n != -1 and s == -1:\n",
    "        df.loc[idxDf]['language'] = 'Nextflow'\n",
    "        if df.loc[idxDf]['XMLaccess'] == 'Free':\n",
    "            nbNextflow += 1\n",
    "            \"\"\"cmd = 'mv ' + path + article + \" \" + 'nextflow/all/'\n",
    "            os.system(cmd)\n",
    "            cmd = 'mv ' + ' article/' + str(idxDf+1) + \"_*.txt\" + \" \" + 'nextflow/'\n",
    "            os.system(cmd)\"\"\"\n",
    "        \n",
    "    elif n == -1 and s != -1:\n",
    "        df.loc[idxDf]['language'] = 'Snakemake'\n",
    "        if df.loc[idxDf]['XMLaccess'] == 'Free':\n",
    "            nbSnakemake += 1\n",
    "            \"\"\"cmd = 'mv ' + path + article + \" \" + 'snakemake/all/'\n",
    "            os.system(cmd)\n",
    "            cmd = 'mv ' + ' article/' + str(idxDf+1) + \"_*.txt\" +  \" \" + 'snakemake/'\n",
    "            os.system(cmd)\"\"\"\n",
    "            \n",
    "            \n",
    "    elif n != -1 and s != -1:\n",
    "        nbBoth += 1\n",
    "        df.loc[idxDf]['language'] = 'Both'\n",
    "        \n",
    "    a.close()\n",
    "print(\"Nombre d'articles decrivant des wf Nextflow : {}\".format(nbNextflow))\n",
    "print(\"Nombre d'articles decrivant des wf Snakemake : {}\".format(nbSnakemake))\n",
    "print(\"Nombre d'articles possédant à la fois Nextflow et Snakemake : {}\".format(nbBoth))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "9fbc14e4",
   "metadata": {},
   "outputs": [],
   "source": [
    "df.to_html('corpus.html',justify='center',index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "278b9fa5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>titre</th>\n",
       "      <th>pmid</th>\n",
       "      <th>pmc</th>\n",
       "      <th>doi</th>\n",
       "      <th>XMLaccess</th>\n",
       "      <th>git</th>\n",
       "      <th>language</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>A bioinformatic-assisted workflow for genome-w...</td>\n",
       "      <td>35979446</td>\n",
       "      <td>9376865</td>\n",
       "      <td>10.1093/nargab/lqac059</td>\n",
       "      <td>Free</td>\n",
       "      <td>github.com/brummetheus/pinc</td>\n",
       "      <td>Nextflow</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>BioUML—towards a universal research platform</td>\n",
       "      <td>35536253</td>\n",
       "      <td>9252820</td>\n",
       "      <td>10.1093/nar/gkac286</td>\n",
       "      <td>Free</td>\n",
       "      <td>github.com/Biosoft-ru/pybiouml , github.com/mo...</td>\n",
       "      <td>Nextflow</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>3</td>\n",
       "      <td>Detection and Validation of Circular DNA Fragm...</td>\n",
       "      <td>35711922</td>\n",
       "      <td>9195511</td>\n",
       "      <td>10.3389/fgene.2022.867018</td>\n",
       "      <td>Free</td>\n",
       "      <td>github.com/snakemake-workflows/cyrcular-calling</td>\n",
       "      <td>Snakemake</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>4</td>\n",
       "      <td>GenErode: a bioinformatics pipeline to investi...</td>\n",
       "      <td>35698034</td>\n",
       "      <td>9195343</td>\n",
       "      <td>10.1186/s12859-022-04757-0</td>\n",
       "      <td>Free</td>\n",
       "      <td>github.com/NBISweden/GenErode , github.com/bro...</td>\n",
       "      <td>Snakemake</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5</td>\n",
       "      <td>nf-gwas-pipeline: A Nextflow Genome-Wide Assoc...</td>\n",
       "      <td>35647481</td>\n",
       "      <td>9137404</td>\n",
       "      <td>10.21105/joss.02957</td>\n",
       "      <td>Free</td>\n",
       "      <td>github.com/montilab/nf-gwas-pipeline</td>\n",
       "      <td>Nextflow</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>106</td>\n",
       "      <td>ATAC2GRN: optimized ATAC-seq and DNase1-seq pi...</td>\n",
       "      <td>30064353</td>\n",
       "      <td>6069842</td>\n",
       "      <td>10.1186/s12864-018-4943-z</td>\n",
       "      <td>Free</td>\n",
       "      <td>github.com/ChioriniLab , github.com/lh3/seqtk</td>\n",
       "      <td>Snakemake</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>107</td>\n",
       "      <td>Sequanix: a dynamic graphical interface for Sn...</td>\n",
       "      <td>29361152</td>\n",
       "      <td>5972652</td>\n",
       "      <td>10.1093/bioinformatics/bty034</td>\n",
       "      <td>Free</td>\n",
       "      <td>github.com/sequana/sequana</td>\n",
       "      <td>Snakemake</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>108</td>\n",
       "      <td>TaxMapper: an analysis tool, reference databas...</td>\n",
       "      <td>29037173</td>\n",
       "      <td>5644092</td>\n",
       "      <td>10.1186/s12864-017-4168-6</td>\n",
       "      <td>Free</td>\n",
       "      <td></td>\n",
       "      <td>Snakemake</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>109</td>\n",
       "      <td>MetaMeta: integrating metagenome analysis tool...</td>\n",
       "      <td>28807044</td>\n",
       "      <td>5557516</td>\n",
       "      <td>10.1186/s40168-017-0318-y</td>\n",
       "      <td>Free</td>\n",
       "      <td>github.com/pirovc/metameta , github.com/biobox...</td>\n",
       "      <td>Snakemake</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>110</td>\n",
       "      <td>An automated workflow for parallel processing ...</td>\n",
       "      <td>26628585</td>\n",
       "      <td>4896369</td>\n",
       "      <td>10.1093/bioinformatics/btv706</td>\n",
       "      <td>Free</td>\n",
       "      <td>github.com/mpicbg-scicomp/snakemake-workflows ...</td>\n",
       "      <td>Snakemake</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>110 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      id                                              titre      pmid  \\\n",
       "0      1  A bioinformatic-assisted workflow for genome-w...  35979446   \n",
       "1      2       BioUML—towards a universal research platform  35536253   \n",
       "2      3  Detection and Validation of Circular DNA Fragm...  35711922   \n",
       "3      4  GenErode: a bioinformatics pipeline to investi...  35698034   \n",
       "4      5  nf-gwas-pipeline: A Nextflow Genome-Wide Assoc...  35647481   \n",
       "..   ...                                                ...       ...   \n",
       "105  106  ATAC2GRN: optimized ATAC-seq and DNase1-seq pi...  30064353   \n",
       "106  107  Sequanix: a dynamic graphical interface for Sn...  29361152   \n",
       "107  108  TaxMapper: an analysis tool, reference databas...  29037173   \n",
       "108  109  MetaMeta: integrating metagenome analysis tool...  28807044   \n",
       "109  110  An automated workflow for parallel processing ...  26628585   \n",
       "\n",
       "         pmc                            doi XMLaccess  \\\n",
       "0    9376865         10.1093/nargab/lqac059      Free   \n",
       "1    9252820            10.1093/nar/gkac286      Free   \n",
       "2    9195511      10.3389/fgene.2022.867018      Free   \n",
       "3    9195343     10.1186/s12859-022-04757-0      Free   \n",
       "4    9137404            10.21105/joss.02957      Free   \n",
       "..       ...                            ...       ...   \n",
       "105  6069842      10.1186/s12864-018-4943-z      Free   \n",
       "106  5972652  10.1093/bioinformatics/bty034      Free   \n",
       "107  5644092      10.1186/s12864-017-4168-6      Free   \n",
       "108  5557516      10.1186/s40168-017-0318-y      Free   \n",
       "109  4896369  10.1093/bioinformatics/btv706      Free   \n",
       "\n",
       "                                                   git   language  \n",
       "0                          github.com/brummetheus/pinc   Nextflow  \n",
       "1    github.com/Biosoft-ru/pybiouml , github.com/mo...   Nextflow  \n",
       "2      github.com/snakemake-workflows/cyrcular-calling  Snakemake  \n",
       "3    github.com/NBISweden/GenErode , github.com/bro...  Snakemake  \n",
       "4                 github.com/montilab/nf-gwas-pipeline   Nextflow  \n",
       "..                                                 ...        ...  \n",
       "105      github.com/ChioriniLab , github.com/lh3/seqtk  Snakemake  \n",
       "106                         github.com/sequana/sequana  Snakemake  \n",
       "107                                                     Snakemake  \n",
       "108  github.com/pirovc/metameta , github.com/biobox...  Snakemake  \n",
       "109  github.com/mpicbg-scicomp/snakemake-workflows ...  Snakemake  \n",
       "\n",
       "[110 rows x 8 columns]"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "770fbebc",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.12"
  },
  "vscode": {
   "interpreter": {
    "hash": "853d8cc187a79f991de8eabec73c5211ab1440896105dc73dfdee74aa5462890"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
